compiler/circle-input-names/src/circle-input-names.test.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+#include <cstring>
+#include <fstream>
+#include <vector>
+#include <regex>
+
+class circle_input_names_test : public ::testing::Test
+{
+protected:
+  void SetUp(void) override;
+  std::vector<std::string> get_input_names_of(std::string op);
```

Comment:
two function name has different convention
```suggestion
  void setup(void) override;
  std::vector<std::string> get_input_names_of(std::string op);
```
or
```suggestion
  void SetUp(void) override;
  std::vector<std::string> GetInputNamesOf(std::string op);
```
?
my preference is the first.

Reply:
The first one is virtual function of GTest's fixture class, so it follows _came case_.
However, in ONE, it seems that it follows _snake case_. 
Which one should I follow it? 🤔

Reply:
I would be better to use same convention, for this second one is preferred.

Reply:
Instead of forcing the one convention, I added comments of each member function for explanation of different conventions.